libflux/Cargo.toml

```diff
@@ -16,6 +16,7 @@ wasm-bindgen = { version = "0.2", features = ["serde-serialize"] }
 chrono = { version = "0.4", features = ["serde"] }
 regex = "1"
 maplit = "1.0.2"
+flatbuffers = "0.6.0"
```

Comment:
Does this mean we depend on some Rust runtime code for flatbuffers?

Reply:
Yes, there is some runtime code that flatbuffers uses for both Rust and Go.